Client-Side/adoptForm/adoptForm.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>AdoptADog</title>
+    <link rel="stylesheet" href="../navbar/navbar.css">
+    <link rel="stylesheet" href="adoptForm.css">
+</head>
+<body>
+    <script type="text/javascript" src="../navbar/navbar.js"></script>
+
+    <script>loadNavbar();</script>
+
+    <main class="mainBody">
+        <form action="" class="adoptionForm">
+            <header>Take the first step to take your<br>fluffy friend home!</header>
+            <a class="sliderLink" href="">What is required of me?</a><br><br>
```

Comment:
I think <a> is more for a hyperlink. Perhaps use a button with "text-decoration: underline" and "background: none"?

Reply:
Made the change